src/index.js

```diff
@@ -11,6 +11,7 @@ const rules = {
   "no-renamed-translation-import": noRenamedTranslationImport,
   "gql-objects": gqlObjects,
   "gql-operation-name": gqlOperationName,
+  "gql-name-match": gqlNameMatch,
```

Comment:
Make sure to all change this based on wtv name you select from the first comment. 

Once the final review is done, we will also need to run `yarn build` and commit those changes/push so that it's all compiled for usage.